src/Umbraco.Core/Manifest/PackageManifest.cs

```diff
@@ -10,6 +10,12 @@ namespace Umbraco.Cms.Core.Manifest
     [DataContract]
     public class PackageManifest
     {
+        [DataMember(Name = "name", IsRequired = true)]
+        public string PackageName { get; set; }
+
+        [DataMember(Name = "packageView", IsRequired = true)]
+        public string PackageView { get; set; }
```

Comment:
I see no reason these are required?

IMO we should aim for not have this as a breaking change in the `package.manifest`.

Reply:
> I see no reason these are required?

No, packageView required is a mistake.